lib/beacon_api/controllers/v1/beacon_controller.ex

```diff
@@ -1,8 +1,37 @@
 defmodule BeaconApi.V1.BeaconController do
   alias BeaconApi.ErrorController
+
+  alias BeaconApi.Schemas.{
+    InternalErrorStateRootResponse,
+    InvalidStateRootResponse,
+    StateNotFoundStateRootResponse,
+    SuccessStateRootResponse
+  }
+
   alias LambdaEthereumConsensus.ForkChoice
   alias LambdaEthereumConsensus.Store.BlockStore
   use BeaconApi, :controller
+  use OpenApiSpex.ControllerSpecs
+
+  operation(:get_state_root,
```

Comment:
There is already a spec provided by the ethereum foundation: https://ethereum.github.io/beacon-APIs/ . We should use this as our spec